ding/utils/lock_helper.py

```diff
@@ -118,3 +118,13 @@ def get_file_lock(name: str, op: str) -> None:
             except Exception as e:
                 pass
         return FcntlContext(lock_name)
+
+
+def synchronized(func):
```

Comment:
for decorator function, you need to add `functools.wraps` to maintain origin function's information

Reply:
Done